### PR TITLE
Remove shell scripts from CI

### PR DESCRIPTION
**Pull request by hesa**
With this commit we remove the shell scripts from CI. Too many failures have been caused by these. Keeping the shell scripts for local use.

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -38,11 +38,6 @@ jobs:
       run: python3 setup.py build
     - name: Install
       run: python3 setup.py install
-    - name: Test
-      run: |
-        cd tests
-        JUNIT_REPORT=1 ./all.sh
```
Comment (jens-erdmann) [inline .github/workflows/push.yml:44]: Instead of removing it entirely I would rather have this replaced with the pytest part.

**Review by jens-erdmann — APPROVED** [review]
LGTM

File: .github/workflows/push.yml [hunk 2]
```diff
@@ -40,9 +40,7 @@ jobs:
       run: python3 setup.py install
     - name: Test
       run: |
```
Comment (jens-erdmann) [inline .github/workflows/push.yml:42]: Small nit: The command could replace `|` here and it would be all in one line.

Comment (hesa) [reply .github/workflows/push.yml:42]: Used the pattern which is there in the file since before.

Comment (hesa) [reply .github/workflows/push.yml:42]: Jens, Konrad: think we can merge this now

Comment (jens-erdmann) [reply .github/workflows/push.yml:42]: Do we actually have to remove it after #151 ? Can you check your other PRs?
I would rather gradually replace bash tests with pytest than lose this coverage.

Comment (priv-kweihmann) [reply .github/workflows/push.yml:42]: I think after a combination of #162 and #151 we need to remove it anyway

Comment (hesa) [reply .github/workflows/push.yml:42]: OK. Let's close the PR then?

Comment (priv-kweihmann) [reply .github/workflows/push.yml:42]: Yes, pls

Comment (jens-erdmann) [reply .github/workflows/push.yml:42]: I think, this is ready to be merged.

Comment (jens-erdmann) [reply .github/workflows/push.yml:42]: @priv-kweihmann final thoughts?

Comment (priv-kweihmann) [reply .github/workflows/push.yml:42]: lgtm

**Pull request merged**